torch/csrc/jit/serialization/import_source.cpp

```diff
@@ -247,6 +247,68 @@ struct SourceImporterImpl : public Resolver,
         nullptr);
   }
 
+  // Extract a dictionary containing ignored argument named for an interface
+  // whose AST is class_def. This is stored as a class attributes named
+  // "__ignored_argument_names__".
+  InterfaceType::InterfaceIgnoredArgsType getIgnoredArgumentNames(
+      const ClassDef& class_def) {
+    InterfaceType::InterfaceIgnoredArgsType ignored_argument_names;
+    for (const Stmt& stmt : class_def.body()) {
+      // Look for Assign.
+      if (stmt.kind() == TK_ASSIGN) {
+        auto assign = Assign(stmt);
+        auto var = Var(assign.lhs());
+
+        // The LHS must be a Var named "__ignored_argument_names__".
+        if (var.name().name() == "__ignored_argument_names__") {
```

Comment:
Can there be multiple `__ignored_argument_names__`? if not, we can exit this method early after finding one.

Reply:
Added early return.